package.json

```diff
@@ -7,13 +7,22 @@
     "dev": "vite",
     "build": "vite build",
     "preview": "vite preview",
-    "deploy":"vite build && git push origin --delete gh-pages && git subtree push --prefix dist origin gh-pages"
+    "deploy": "vite build && git push origin --delete gh-pages && git subtree push --prefix dist origin gh-pages"
   },
   "dependencies": {
-    "@resonai/vera-sdk": "^0.0.10"
+    "//@resonai/vera-sdk": "^0.0.11",
+    "@resonai/vera-sdk": "file:../resonai2/irocket/client/web/verasdk/"
```

Comment:
Revert?
Also - this might be a better option to help make local development easier:
https://classic.yarnpkg.com/en/docs/cli/link

(I think it won't require the removal of the package every time)

Reply:
I'm reverting this in a later PR.
Not sure I got the 2nd part of your comment